README.md

```diff
@@ -27,13 +27,17 @@ The Rust reference implementation of the COMIT protocol (comit-rs) implements at
 This repository is a cargo workspace:
 
 - `cnd`: implementation of the comit-network daemon
+- `nectar`: implementation of a simple automated trader for demo purposes
+- `comit-lib`: shared library used by `cnd` and `nectar` for swap execution
 
 `cnd` is [released](https://github.com/comit-network/comit-rs/releases) as binary.
 
 ## Setup build environment
 
 All you need is ~love~ rust: `curl https://sh.rustup.rs -sSf | sh`
 
+For building nectar you might have to install open ssl dependencies: `sudo apt-get install libssl-dev`
```

Comment:
```suggestion
For building nectar you might have to install openssl dependencies: `sudo apt-get install libssl-dev`
```

Reply:
I'd propose an alternative solution to this problem: https://github.com/comit-network/comit-rs/pull/3209